composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Simple file-based plugin activation control for WordPress",
     "require": {
         "php": "^5.6 || ^7.0",
-        "symfony/yaml": "^3.4 || ^4",
+        "symfony/yaml": "^5.2",
```

Comment:
This version constraint isn't compatible with the minimum PHP here (5.6) as `symfony/yaml` v5 requires PHP 7.2+.

Would you please extend it similar to how it was rather than replacing it?

Reply:
Argh,

@aaemnnosttv Do you mind if I take up the idea of your project and do it mainly for myself, put it on my personal github and put a link on your repository (I like the idea very much and would like to set this up for my Bedrock sites and maintain it "myself")

Reply:
You don't need my permission to do that 😄  Of course you could fork the repo and maintain your own version, you'd just have to install it differently. No objections from me though 👍